content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -17,20 +17,25 @@ update your firewall with the port for that new database endpoint.
 
 ## Ports and port ranges used by Redis Enterprise Software
 
-| Protocol | Port | Description |
-|------------|-----------------|-----------------|
-| ICMP | * | For connectivity checking between nodes |
-| TCP | 1968 | Proxy traffic (Internal use) |
-| TCP | 3333, 3334, 3335, 3336, 3337, 3338, 3339, 36379, 36380 | Cluster traffic (Internal use) |
-| TCP | 8001 | Traffic from application to RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
-| TCP | 8443 | Secure (HTTPS) access to the management web UI |
-| TCP | 8444, 9080 | For nginx <-> cnm_http/cm traffic (Internal use) |
-| TCP | 9081 | For CRDB management (Internal use) |
-| TCP | 8070, 8071 | For metrics exported and managed by nginx |
-| TCP | 9443 (Recommended), [8080](#turning-off-http-support) | REST API traffic, including cluster management and node bootstrap |
-| TCP | 10000-19999 | Database traffic |
-| TCP | 20000-29999 | Database shards traffic (Internal use) |
-| UDP | 53, 5353 | DNS/mDNS traffic (Internal use) |
+| Protocol | Port | Connection Source | Description |
```

Comment:
I think "Use" is a better title... @bmansheim - what do you think?

Reply:
@Paz-Yanover I actually like "Connection Source"